hudi-integ-test/src/test/java/org/apache/hudi/integ/ITTestHoodieSanity.java

```diff
@@ -193,19 +193,6 @@ public void testRunHoodieJavaApp(String command, String hiveTableName, String ta
       assertEquals(80, Integer.parseInt(stdOutErr.getLeft().substring(stdOutErr.getLeft().lastIndexOf("\n")).trim()),
           "Expecting 80 rows to be present in the snapshot table");
     }
-
-    // Make the HDFS dataset non-hoodie and run the same query; Checks for interoperability with non-hoodie tables
-    // Delete Hoodie directory to make it non-hoodie dataset
-    executeCommandStringInDocker(ADHOC_1_CONTAINER, "hdfs dfs -rm -r " + hdfsPath + "/.hoodie", true);
-
```

Comment:
We need meta table when we query hive now, this case doesn't fit anymore.

Reply:
> meta table when we query hive now

For meta table do you mean the metadata table?